app/src/main/java/org/p2p/wallet/home/ui/crypto/MyCryptoPresenter.kt

```diff
@@ -126,6 +129,7 @@ class MyCryptoPresenter(
 
         return tokens
             .mapNotNull(Token.Active::totalInUsd)
+            .filter { it.isMoreThan(MINIMAL_DUST_FOR_BALANCE) }
```

Comment:
looks like domain logic
do you want maybe extract this getUserBalance to the interactor?